explanation/maintenance-mode.rst

```diff
@@ -0,0 +1,65 @@
+Maintenance Mode
+================
+
+Overview
+--------
+
+Maintenance mode provides a safeguard for operators performing critical operations 
+that could potentially disrupt services or result in data loss. Before executing 
+any high-risk actions on a node, operators should enable maintenance mode to 
+minimize impact and ensure system stability.
+
+States of the Node
```

Comment:
I saw this output from `sunbeam cluster list`

```shell
$ sunbeam cluster list
┏━━━━━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━┓
┃ Node         ┃ Machine ┃ Compute ┃ Control ┃ Storage ┃
┡━━━━━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━┩
│ xxx.xxx.xxx.xxx │ running │ active  │ active  │ active  │
└──────────────┴─────────┴─────────┴─────────┴─────────┘
```

I think it will be nice to dedicate a separate page to define the states. And then mention maintenance state here with a link refer back to the page for states. Otherwise, it might be confusing for the reader (e.g. what is a `running` state, why it's not mentioned here etc)

Reply:
Added a note under it. I feel the content of states can't not be a separate document. Let me know if you still want me to separate it.